_posts/en/newsletters/2024-06-28-newsletter.md

```diff
@@ -0,0 +1,122 @@
+  we can look at what percentages of wealth distributions in the
+  Alice-Bob channel and the Bob-Carol channel would allow that payment
+  to succeed.  If the Alice-Bob channel has capacity of several BTC,
+  most possible wealth distributions would allow the payment to succeed.
+  If the Bob-Carol channel has a capacity just barely over 1 BTC, then
+  most possible wealth distributions would prevent the payment from
+  succeeding.  This can be used to calculate the overall likelihood
+  of the feasibility of a payment of 1 BTC from Alice to Carol.
+
+  The likelihood of feasibility makes it clear that many LN payments
+  that naively seem possible will not succeed in practice.  It also
+  provides a useful basis for comparison: Pickhardt provides a graph for
+  a simulated network that compares the likelihood of feasibility to the
+  likelihood of a minimum cost flow solver making a successful payment,
+  showing the solver succeeding most of the time a payment is possible.
+  In a [reply][pickhardt feasible2], he also describes how the
+  likelihood metric could be used by wallets and business software to
+  automatically make some intelligent decisions on behalf of its users.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.2rc1][] is a release candidate for a maintenance
+  version of Bitcoin Core's older release series.
+
+<!-- FIXME:harding to update Thursday -->
+
+## Notable code and documentation changes
+
+_Notable recent changes in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #29575][] net_processing: make any misbehavior trigger immediate discouragement
+
+- [Bitcoin Core #28984][] Cluster size 2 package rbf
+
+- [Core Lightning #7388][] no longer allow creation of old
```

Comment:
```suggestion
- [Core Lightning #7388][] no longer allows the creation of old
```

Reply:
(Ignoring; this isn't text for the newsletter, it's from the CLN commit message.  I include some text for each unwritten PR so that @Gustavojfe can tell if I've typed the wrong PR number or something.)